In [6]:
import pandas as pd

# Chargement et Exploration des Données

In [7]:
data = pd.read_csv('../data/Chocolate Sales.csv')
data.head()

Sales Person    Country              Product       Date    Amount  \
0  Jehu Rudeforth         UK      Mint Chip Choco  04-Jan-22   $5,320    
1     Van Tuxwell      India        85% Dark Bars  01-Aug-22   $7,896    
2    Gigi Bohling      India  Peanut Butter Cubes  07-Jul-22   $4,501    
3    Jan Morforth  Australia  Peanut Butter Cubes  27-Apr-22  $12,726    
4  Jehu Rudeforth         UK  Peanut Butter Cubes  24-Feb-22  $13,685    

   Boxes Shipped  
0            180  
1             94  
2             91  
3            342  
4            184

In [8]:
data.tail()

Sales Person    Country              Product       Date   Amount  \
1089  Karlen McCaffrey  Australia  Spicy Special Slims  17-May-22  $4,410    
1090    Jehu Rudeforth        USA           White Choc  07-Jun-22  $6,559    
1091      Ches Bonnell     Canada  Organic Choco Syrup  26-Jul-22    $574    
1092    Dotty Strutley      India              Eclairs  28-Jul-22  $2,086    
1093  Karlen McCaffrey      India       70% Dark Bites  23-May-22  $5,075    

      Boxes Shipped  
1089            323  
1090            119  
1091            217  
1092            384  
1093            344

### Description du dataset

Variables
- Sales Person : Nom et Prénom des vendeurs — format [Nom Prénom]
- Country : Pays — format [Pays]
- Product : Nom du Chocolat - format [Nom]
- Date : Date de vente - format [jj-Mon-AA]
- Amount : Prix de vente - format [$prix]
- Box Shipped : Nombre de boîtes expédiées — format [int]


In [9]:
data.shape

(1094, 6)

Il y a 1094 ventes


### Question d'Analyse

- Dans quel pays avons-nous le plus de vente ?
- Qui sont les plus grands vendeurs internationaux ?
- Quels sont les chocolats qui connaissent un plus grand succès ?
- Quels vendeurs ont les meilleurs bénéfices ?
- Y a-t-il une période dans l'année où il y a une hausse des ventes de chocolats ?
- Y a-t-il des inégalités dans la quantité de chocolats expédiés à travers le monde ?